grafana/Makefile

```diff
@@ -36,7 +36,7 @@ ifeq ($(ARCH),amd64)
 	BASEIMAGE?=busybox
 	CC=gcc
 endif
-ifeq ($(ARCH),arm)
+ifeq ($(ARCH),armhfrm)
```

Comment:
@piosz Indeed an invalid change. I can't remember seeing that when I reviewed, probably came later in the review process.